generate.py

```diff
@@ -37,6 +38,9 @@ def compile(src: Path, libdir: Path):
         cxxflags_default = '-O2 -std=c++14 -Wall -Wextra -Werror -Wno-unused-result'
         if platform.system() == 'Darwin':
             cxxflags_default += ' -Wl,-stack_size,0x10000000'  # 256MB
+        if platform.system() == 'Windows':
+            cxxflags_default += ' -Wl,-stack,0x10000000'  # 256MB
+            cxxflags_default += ' -D__USE_MINGW_ANSI_STDIO' # avoid using MinGW's "unique" stdio, which doesn't recognize %lld
```

Comment:
```suggestion
            cxxflags_default += ' -D__USE_MINGW_ANSI_STDIO' # avoid using MinGW's "unique" stdio, which doesn't recognize %lld
```

これつけると%lldが動くようになるなら、testlibの%lldをI64に置き換える修正は必要ですか？

Reply:
STDIOとある通りなのかprintfとかしか直してくれなくてtestlibのquitf(-> vsprintf)はダメでした
どうしようもなかったのでquitfだけはI64を使う必要がありました